src/graphics/opengl/RendererGL.cpp

```diff
@@ -1089,8 +1089,8 @@ namespace Graphics {
 		if (desc.colorFormat != TEXTURE_NONE && desc.depthFormat != TEXTURE_NONE && !rt->CheckStatus()) {
 			GLuint status = glCheckFramebufferStatus(GL_FRAMEBUFFER);
 			Log::Error("Unable to create complete render target. (Error: {})\n"
-				"Does your graphics driver support multisample anti-aliasing?\n"
-				"If this issue persists, try setting AntiAliasingMode=0 in your config file.\n",
+					   "Does your graphics driver support multisample anti-aliasing?\n"
+					   "If this issue persists, try setting AntiAliasingMode=0 in your config file.\n",
```

Comment:
Sigh... clang-format constantly tries to squirrel its way into aligning arguments or parameters in an incorrect way. Nothing you need to address here, but I'll have to review the clang-format file _again_, for the Nth time.